aws/websites/metrics.pytorch.org/files/dashboards/lambda_status.json

```diff
@@ -0,0 +1,782 @@
+{
```

Comment:
It's the JSON definition from the UI itself (i.e when you make a dashboard it is just generating this JSON behind the scenes), manually exported and saved to a file. It'd be nice if grafana was able to tell you about resources created in the UI and not via provisioning since there's no easy way to tell AFAIK.

I added some docs explaining this since it applies to any dashboard someone might create